winit-core/src/window.rs

```diff
@@ -69,53 +36,9 @@ pub struct WindowAttributes {
     pub cursor: Cursor,
     pub(crate) parent_window: Option<SendSyncRawWindowHandle>,
     pub fullscreen: Option<Fullscreen>,
-    // Platform-specific configuration.
-    #[allow(dead_code)]
-    pub(crate) platform_specific: PlatformSpecificWindowAttributes,
+    pub platform: Option<Box<dyn PlatformWindowAttributes>>,
```

Comment:
> Also, the `WindowAttributes` and the `platform` part on it I'm not that found of, it does kind of what it should, but I feel like there could be a better way...

I think what you ended up with here is fine, though I'd like it to be done in a separate PR. I can do it if you want me to?

Reply:
We can just keep it as separate commit.

Reply:
Fine with it given that the PR isn't squashed.